better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
```

Comment:
@mattheu good point.. so making use of `$template`:

````php
function get_template( $template, $args ) {
    $path = apply_filters( 'bfa_get_template', __DIR__ . '/templates/'. $template, $args ); 
    if ( file_exists( $path ) ) {
        ob_start();
        include $path;
        return ob_get_clean();
    }
}
````
Questions:
- I'm not sure what `$args` would do?
- Should I use `plugin_dir_path( __FILE__ )` instead of `__DIR__`?